adapters/33across/33across.go

```diff
@@ -38,39 +51,62 @@ func (a *TtxAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters
 	var errs []error
 	var adapterRequests []*adapters.RequestData
 
-	adapterReq, errors := a.makeRequest(request)
-	if adapterReq != nil {
-		adapterRequests = append(adapterRequests, adapterReq)
+	// Break up multi-imp request into multiple external requests since we don't
+	// support SRA in our exchange server
+	for i := 0; i < len(request.Imp); i++ {
+		adapterReq, err := a.makeRequest(request, request.Imp[i])
+		if adapterReq != nil {
```

Comment:
It's more common to check for errors before proceeding, rather than checking for nil. Ideally should be closer to:
```
if adapterReq, err := a.makeRequest(request, request.Imp[i]); err != nil {
  adapterRequests = append(adapterRequests, adapterReq)
} else {
  errs = append(errs, err)
}


```

Reply:
assuming you mean `err == nil` check in the if statement?

Reply:
Ha. Yes. Absolutely. :)